opengemini_client/models.py

```diff
@@ -62,9 +63,44 @@ class RpConfig:
     index_duration: str
 
 
+class PrecisionType(Enum):
```

Comment:
https://github.com/openGemini/opengemini-client-go/pull/69 change PrecisionType to Precision , plz synchronize as the pr